setup.py

```diff
@@ -39,7 +39,7 @@
         "requests",
         "rich",
         "ruamel.yaml",
-        "sqlglot>=11.4.4",
+        "sqlglot~=11.5.1",
```

Comment:
what does this mean?

Reply:
Compatible releases: https://peps.python.org/pep-0440/#compatible-release